triagebot.toml

```
@@ -122,6 +122,17 @@ issues).
 """
 label = "O-apple"
 
+[ping.rust-for-linux]
+alias = ["rfl"]
+message = """\
+Hey Rust for Linux group! It looks like something broke the Rust for Linux integration.
+Could you try to take a look?
+In case it's useful, here are some [instructions] for tackling these sorts of issues.
```

I think it's better to stay generic in the message, like it's done for the `apple` ping group above.

```suggestion
Hey Rust for Linux group! This issue or PR could use some Rust for Linux guidance.
Could you try to take a look? Thanks <3

In case it's useful, here are some [instructions] for tackling these sorts of issues.
```

---

I'd say that it's a bit different, because this ping group is not about helping rustc developers resolve issues with RfL (like it is for Apple, Windows etc.), but mostly about notifying RfL people that something in rustc broke.

I think that for now we should default to telling contributors "your PR broke RfL and the corresponding people should know about it" rather than "your PR broke RfL, let's ask these people how to avoid that".

---

I that case would it be better to use a more descriptive name like `rfl-ci-break` or something else ? So that contributors don't use it for what it isn't meant to be used for.

---

Maybe :man_shrugging: I'm not sure what would rustc contributors use it for anyhow, as they probably don't interact with RfL at all. Adding a comment makes sense, I'll add it.